file-formats/sfpf/type/zif_aff_sfpf_v1.intf.abap

```diff
@@ -0,0 +1,34 @@
+INTERFACE zif_aff_sfpf_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Object Type SFPF</p>
+    "! Object type SFPF
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version  TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header          TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Name</p>
+      "! Name
+      "! $required
+      name            TYPE c LENGTH 30,
+      "! <p class="shorttext">Description</p>
+      "! Description
+      "! $required
+      description     TYPE c LENGTH 60,
+      "! <p class="shorttext">Dataprovider</p>
+      "! Dataprovider
+      "! $required
+      dataprovider    TYPE c LENGTH 40,
+      "! <p class="shorttext">Font Embed</p>
+      "! Font Embed
+      embed           TYPE abap_bool,
+      "! <p class="shorttext">Mirror in RTL</p>
+      "! Mirror in RTL
+      mirroring       TYPE abap_bool,
```

Comment:
Can these be more meaningful names? Usually we also recommend the same name for the field as you choose as description for the title of the field (text within `<p class="shorttext">...</p>`)

Reply:
Descriptions are adjusted to be more meaningful.